### ISSUE TITLE
Is it time to get rid of Tomcat and use a packaged application server?

### ISSUE THREAD

**Issue by ibnesayeed** [iipc/openwayback#343]
In one of the telecons we talked about not using Tomcat for the CDX Server. Should we also do the same for the OpenWayback. I think it will make installation OWB a lot easier.

**Comment by ldko** [comment]
Hi Sawood,
Is this something you have looked into implementing?

**Comment by ibnesayeed** [comment]
@ldko, no, I didn't. However, a while ago we talked about not using Tomcat in CDX Server and thinking about not using it in OWB 3.x either. Additionally, the proposal was to utilize a YAML style config file rather than verbose Spring XMLs.

**Comment by ldko** [comment]
Right, I remember talking about those things. Development on OWB 3.x has stalled. I am not sure whether that is going to pick up again unless someone wants to take up a lot of development on this project. Pretty much we will move forward with what people want to contribute work on at this point.
